### PR TITLE
[TASK] Modified who can be listed

### PR DESCRIPTION
**Pull request by mabolek**
Included Association employees and persons appointed by the TYPO3 Association Board

### CODE REVIEW

**Comment by linawolf** [comment]
@mabolek this PR is open since may, what should happen to it?

**Comment by mabolek** [comment]
@linawolf It should still be merged. I'm still waiting for a review from @luisasofie to make sure we don't enact new policies alone. I'll send her a reminder.

**Review by luisasofie — APPROVED** [review]
changes approved

**Pull request merged**
